### ISSUE TITLE
dont depend on nvim-treesitter

### ISSUE THREAD

**Issue by teto** [nvim-orgmode/orgmode#567]
nvim-treesitter installs tons of queries ([nvim-treesitter/nvim-treesitter@master/queries](https://github.com/nvim-treesitter/nvim-treesitter/tree/master/queries?rgh-link-date=2023-05-30T12%3A09%3A30Z)) that are picked up by neovim even though we sometimes dont want those queries (when installing the upstream plugin with those queries for instance). 
I am trying to get rid of the nvim-treesitter dependency since it caused issues on my setup. orgmode is the only plugin that causes problem for me as it needs nvim-treesitter in https://github.com/nvim-orgmode/orgmode/blob/3d6acc695184a964741ef00dfe7c45028585215e/lua/orgmode/utils/init.lua#L438 . If this could be implemented using neovim core instead it would be great.

**Comment by teto** [comment]
ah yes my bad I dont know I missed the other instances, I had just found one. I leave this open since I still think it makes sense. Will try to see later how I can work around this issue.

**Comment by schultzter** [comment]
Yeah, I really feel like treesitter should be an optional dependancy! I'm not a programmer, I'm using Neovim for notes and log files and config files so depending on treesitter is really overwhelming for me.

**Comment by kristijanhusak** [comment]
I set up a branch that does not have a dependency on nvim-treesitter: https://github.com/nvim-orgmode/orgmode/tree/feat/no-nvim-ts

I'm still giving it a test, works fine for me on Linux and Windows 11. If anyone is using Mac i'd appreciate if they would give it a test.

**Comment by schultzter** [comment]
Working perfectly for me. Though I just discovered another plugin with an excessive dependency on treesitter (so I disabled that plugin since I can live without it). I guess the underlying issue is treesitter needs to be more accommodating of network conditions so that it _just works_.
